RedHatInsights/vulnerability-ui
# fix: Fix crash on export kebab open

Fixes [VULN-2361](https://issues.redhat.com/browse/VULN-2361).

This replaces DropdownItem with button, because of some bug in PatternFly which crashes the page, there should be no other change in functionality or design.

Also removes some unused components:
- kebabItemDownloadJSON
- kebabItemDownloadCSV
- BaseKebab
- DownloadReportKebab

How to test:
Go to https://stage.foo.redhat.com:1337/beta/insights/vulnerability/cves and try to export table data to PDF, page should not crash in the process.
